schemas/PositiveFiniteAmount.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "PositiveFiniteAmount",
+  "title": "PositiveFiniteAmount",
+  "description": "A string representation of a positive, finite number",
+  "type": "string",
+  "pattern": "/^[+]?([1-9][0-9]*(?:[.][0-9]*)?|0*.0*[1-9][0-9]*)(?:[eE][+-][0-9]+)?$/"
```

Comment:
That will validate against `0` and not against `0.1`. I'm all for simplifying the regex, but I haven't found anything shorter that does what I need. On the other hand, we could allow `0` here, since the amount is validated here: https://github.com/interledger/five-bells-connector/blob/master/src/models/quote.js#L158. That may be easiest.